versions.tf

```
@@ -0,0 +1,19 @@
+terraform {
+  required_version = ">= 1.5.5"

```

É interessante manter as [garantias de compatibilidade da v1](https://www.terraform.io/language/v1-compatibility-promises) do Terraform.

```suggestion
  required_version = "~> 1.5"
```